pawno/include/PPC_PlayerCommands.inc

```diff
@@ -2369,30 +2405,33 @@ COMMAND:engine(playerid, params[])
 	// Check if the player has logged in
 	if (APlayerData[playerid][LoggedIn] == true)
 	{
-		// Check if the player's admin-level is at least 1
-		if (APlayerData[playerid][PlayerLevel] >= 1)
+		//check if the player its the driver of the car, don't let the passanger to control the engine
+		if(GetPlayerState(playerid) == PLAYER_STATE_DRIVER)
 		{
 			// Get the player's vehicle
 			vehicleid = GetPlayerVehicleID(playerid);
 
-			// Get the current status of the vehicle
-			GetVehicleParamsEx(vehicleid, engine, lights, alarm, doors, bonnet, boot, objective);
-
 			// Check if the player is inside a vehicle
 			if (vehicleid != 0)
```

Comment:
No need for this check as we already have checked if the player is a driver of a vehicle.